sidechain/fork-tree/src/lib.rs

```diff
@@ -0,0 +1,1688 @@
+// This file is part of Substrate.
```

Comment:
Any changes to the original file here? If not, we should probably say that it is identical.

Reply:
There is a few changes to the file but not much. One is our change regarding std and I did have to change one other small detail

Reply:
Alright, I think we should add a comment there so that we know our adjustments when we integrate upstream changes.